extension/background/youtubei.js

```diff
@@ -54,3 +54,83 @@ function parseTimestamp(ts) {
     const hours = parseInt(parts[2]) || 0
     return secs + (60 * mins) + (60 * 60 * hours)
 }
+
+// time-comments related code below
+
+const MAX_COMMENT_PAGES = 5
+const MAX_COMMENTS = 100
+
+const INNERTUBE_API_KEY = "AIzaSyAO_FJ2SlqU8Q4STEHLGCilw_Y9_11qcW8"
+const INNERTUBE_CLIENT_NAME = "WEB"
+
+export async function fetchComments(videoId) {
+    const videoResponse = await fetchVideo(videoId)
```

Comment:
We have already fetched the video in `fetchChapters`. There is no need in the second HTTP request. We should use the previous result.

Reply:
Okay, I'll reuse the earlier fetched data.